config/application.rb

```diff
@@ -11,6 +11,8 @@
 require 'action_mailer/railtie'
 require 'action_view/railtie'
 # require 'action_cable/engine'
+require 'jquery-rails'
```

Comment:
What are these used for?

Reply:
They used to come in via rails-admin, we need them for some of the form interactions I guess, since the page wasn't rendering when they were no longer included.

Reply:
It does indeed seem to be required by nested_form_fields' links for removing field groups.